Contributing.md

```diff
@@ -1,42 +1,83 @@
 # Contribute to this documentation
 
-Thank you for your interest in Office Scripts documentation!
+Thank you for your interest in our documentation! The best ways to help make this documentation better for everyone are to:
 
-* [Ways to contribute](#ways-to-contribute)
-* [Using Git and GitHub](#using-git-and-github)
-* [Adding samples](#adding-samples)
-* [FAQ](#faq)
-* [More resources](#more-resources)
+* [Report issues or request additional samples.](#report-issues)
+* [Add or edit new samples.](#add-or-edit-samples)
 
-## Ways to contribute
+## Report issues
 
-Here are some ways you can contribute to this documentation:
+Most of this repo's content is generated from [type definition files](https://github.com/DefinitelyTyped/DefinitelyTyped). As such, corrections here are likely to be overwritten. Please report issues or make requests for additional documentation through [GitHub Issues](https://github.com/OfficeDev/office-scripts-docs-reference/issues). To do this, go to the **Feedback** section at the bottom of the affected article, then select **This page** to create a GitHub issue. Alternatively, create a new issue directly on [GitHub](https://github.com/OfficeDev/office-scripts-docs-reference/issues/new).
```

Comment:
Is this true for Office Scripts that the type definition files are in DefinitelyTyped?

Reply:
Good catch. They are not.